package.json

```diff
@@ -21,6 +21,10 @@
     "eslint-plugin-react": "^7.23.2"
   },
   "dependencies": {
+    "chalk": "^4.1.1",
+    "fs": "0.0.1-security",
+    "os": "^0.1.1",
+    "path": "^0.12.7",
```

Comment:
It's built-in packages, looks like you import fake packages from npm instead of built-in
![8552](https://user-images.githubusercontent.com/4405297/120109280-62eb8700-c171-11eb-9f83-860975fe52f9.gif)